modular_skyrat/modules/modular_vending/code/vending.dm

```diff
@@ -13,7 +15,12 @@
 		premium += skyrat_premium
 	if(skyrat_contraband)
 		contraband += skyrat_contraband
-
+	if(skyrat_amount_override)
+		for (var/item in products)
+			products[item] = skyrat_amount_override
+		for (var/item in premium)
+			premium[item] = skyrat_amount_override
```

Comment:
```suggestion
			if(premium[item] < skyrat_amount_override)
				premium[item] = skyrat_amount_override
```

Reply:
These two changes will prevent already above 5 items from getting reduced down to 5, ie. Families clothing, which is meant to be available in bulk.

Reply:
Yup, I do have that in my local repo already, I just haven't pushed it yet!